src/sonic-config-engine/tests/test_cfggen_platformJson.py

```diff
@@ -85,3 +91,10 @@ def test_platform_json_all_ethernet_interfaces(self):
         output_dict = ast.literal_eval(output.strip())
         expected = ast.literal_eval(json.dumps(fh_data))
         self.assertDictEqual(output_dict, expected)
+
+    @mock.patch('portconfig.readJson', mock.MagicMock(return_value={INTF_KEY:{}}))
```

Comment:
I am not sure what this file is? Can you please elaborate? Additionally, can you confirm if the format readJson is correct?